tests/rest/rstcheckBearTest.py

```diff
@@ -0,0 +1,10 @@
+from bears.rest.rstcheckBear import rstcheckBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = "test\n====\n"
+bad_file = "test\n==\n"
```

Comment:
Could you add a second good & bad file which has a code block that is good and bad respectively. That is the 'killer' feature of this bear.